doc/MSTP/MSTP.md

```diff
@@ -0,0 +1,1178 @@
+                    	}
+		    }
+		    default 2;
+                    description
+                        "MST hello time";
+                }
+
+                leaf max_age {
+                    must "../../../STP/STP_LIST[keyleaf='GLOBAL']/mode='mst'" {
+                        error-message "Configuration allowed in MST mode";
+                        error-app-tag stp-invalid;
+                    }
+                     type uint8 {
+                    	range "6..40" {
+                        	error-message "max_age value out of range";
+                    	}
+	            }
+                    default 20;
+                    description
+                        "MST max age";
+                }
+
+                leaf forward_delay {
+                    must "../../../STP/STP_LIST[keyleaf='GLOBAL']/mode='mst'" {
+                        error-message "Configuration allowed in MST mode";
+                        error-app-tag stp-invalid;
+                    }
+	            type uint8 {
+                    	range "4..30" {
+                        	error-message "forward_delay value out of range";
+			}
+                    }
+                    default 15;
+                    description
+                        "MST forward delay";
+                }
+	 }
+	 container STP_MST_INST {
+            list STP_MST_INST_LIST {
+                key "instance";
+
+                leaf instance {
+                    type uint16;
+                    description
+                        "Instance identifier";
+                }
+
+                leaf-list vlan {
+                    type string;
+                    description
+                        "Vlan list";
+                }
+
+                leaf bridge_priority {
+                    type uint16 {
+                        range "0..61440" {
+                            error-message "Invalid Bridge Priority value.";
+                        }
+                    }
+                    default 32768;
+                    description
+                        "The manageable component of the Bridge Identifier";
+                }
+            }
+        }
+
+        container STP_MST_PORT {
+            list STP_MST_PORT_LIST {
+                key "inst_id ifname";
+
+                leaf inst_id {
+                    type leafref {
+                        path "../../../STP_MST_INST/STP_MST_INST_LIST/instance";
+                    }
+                    description
+                        "Reference to MST Instance";
+                }
+
+                leaf ifname {
+                    type leafref {
+                        path "../../../STP_PORT/STP_PORT_LIST/ifname";
+                    }
+                    description
+                        "Reference to Ethernet interface or PortChannel";
+                }
+                uses interfaceAttr;
+            }
+        }
+
+	list _STP_MST_INST_TABLE_LIST {
```

Comment:
As discussed in the call please remove "_" before the name of the list.

Reply:
Removed